Update screenshots and steps for new UI architecture (#1234)
astronomer/docs: astro/api-keys.md

> To create an API key for a Deployment:

Reviewer: ```suggestion
```

I suppose delete this, and some of the space around it?

Author: I think Rob has this covered in a separate PR https://github.com/astronomer/docs/pull/1230

1. In the Cloud UI, open your Deployment.

2. Click **API Keys**.

3. Click **+ API Key**:

    ![Add API key button](/img/docs/add-api-key.png)

> 4. Give the key a name and description, then click **Create API Key**:

Reviewer 2: ```suggestion
4. Enter a name and a description for the API key and then click **Create API Key**:
```

## Delete an API key

> 

Author: ```suggestion

If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.

```

1. In the Cloud UI, select a Workspace and then select a Deployment.

2. Click **API Keys**

> 3. Click the **Edit** button next to your API key.

Reviewer 2: ```suggestion
3. Click **Edit** next to your API key.
```

![image](https://user-images.githubusercontent.com/104205257/191503925-329ec366-afb1-4935-b39b-45c65bd4ed8d.png)

    ![Edit API key button](/img/docs/edit-api-key.png)

> 3. Click **Delete API Key**, and then follow the onscreen prompt to finalize the deletion.

Reviewer 2: ```suggestion
3. Click **Delete API Key**, enter `Delete`, and then click **Yes, Continue**.
```

> If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.

Reviewer 2: ```suggestion
If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.
```

This should follow the heading. It's important information users should have BEFORE completing the procedure.

> If you delete an API key, make sure that no existing CI/CD pipelines are using it. Once deleted, an API key and secret cannot be recovered. If you unintentionally delete an API key, create a new one and update any CI/CD workflows that used the deleted API key.

Author: ```suggestion
```
